### PR TITLE
[0-size Tensor No.193] Add 0-size Tensor support for mode margin_ranking_loss

### PR DESCRIPTION
**Pull request by co63oc**
### PR Category

Execute Infrastructure

### PR Types

Improvements 

### Description

复测
193 paddle.nn.functional.margin_ranking_loss
GPU 通过
CPU 不支持float16，其他通过
image

### CODE REVIEW

**Comment by DanielSun11** [comment]
paddleAPITest支持margin_ranking_loss和torch对比。请验证下是否还存在paddle error

**Comment by co63oc** [comment]
> paddleAPITest支持margin_ranking_loss和torch对比。请验证下是否还存在paddle error

已复测，sigmoid_focal_loss 是精度错误

**Comment by DanielSun11** [comment]
> > paddleAPITest支持margin_ranking_loss和torch对比。请验证下是否还存在paddle error
> 
> 已复测，sigmoid_focal_loss 是精度错误


精度错误也需要修改，请检查下默认初始化的值。
请拉取最新的paddleAPITest添加paddleAPITest report中0-size出错的config，并完成paddleapitest accuracy=True 所有case的复测。

**Comment by co63oc** [comment]
> > > paddleAPITest支持margin_ranking_loss和torch对比。请验证下是否还存在paddle error
> > 
> > 
> > 已复测，sigmoid_focal_loss 是精度错误
> 
> 精度错误也需要修改，请检查下默认初始化的值。 请拉取最新的paddleAPITest添加paddleAPITest report中0-size出错的config，并完成paddleapitest accuracy=True 所有case的复测。

有点复杂，先去掉了 sigmoid_focal_loss

**Review by DanielSun11 — APPROVED** [review]
LGTM

**Review by wanghuancoder — APPROVED** [review]
